CHARTER.md

```diff
@@ -4,10 +4,12 @@ The CHERI SIG will work on a strategy for adding a capability based security mod
 
 Background:
 
-CHERI is a technology that extends instruction sets with new architectural features (so-called capabilities) to enable fine-grained memory protection and highly scalable software compartmentalization. CHERI was developed at the University of Cambridge and SRI and includes an instantiation for RISC-V (32- and 64-bit) using the custom opcode space.
+CHERI is a technology that extends instruction sets with new architectural features (so-called capabilities) to enable fine-grained memory protection and highly scalable software compartmentalization. CHERI was developed at the University of Cambridge and SRI and includes an instantiation for RISC-V (32- and 64-bit) using the custom opcode space. Microsoft has also announced an IoT-focused adaptation to be published in 2023.
```

Comment:
I would hold off on mentioning specifications that have not been published yet.

Reply:
Also, if we mention Microsoft's effort, do we also have to mention Arm Morello?

Reply:
Microsoft’s spec will be published in the next few weeks, and I think will represent a pretty major activity for the group. If we don’t want to have it be a charter activity, we might be better off not proceeding with the charter until they’ve released it.

While Arm Morello is obviously amazing and will influence everyone’s thinking, it isn’t a proposal to extend RISC-V. Whereas I think we might well want to standardise a microcontroller variant of CHERI-RISC-V modelled on (and validated by) Microsoft’s work.